### ISSUE TITLE
And, I would like to relate this ticket with https://github.com/International-Data-Spaces-Association/InformationModel/commit/d492bc16e93ca1614a3ea4f7efa594747cf277a7.

### ISSUE THREAD

**Issue by jlangkau** [International-Data-Spaces-Association/InformationModel#294]
And, I would like to relate this ticket with https://github.com/International-Data-Spaces-Association/InformationModel/commit/d492bc16e93ca1614a3ea4f7efa594747cf277a7. 
My suggestion: 
We need to specify how Time is expected in the IDS, either XSD Datatypes (simple) or TemporalInstances (complex types). We should not define time instances, days of week, months etc. by ourselves.

_Originally posted by @sebbader in https://github.com/International-Data-Spaces-Association/InformationModel/issues/273#issuecomment-655010415_

**Comment by jlangkau** [comment]
> And, I would like to relate this ticket with [d492bc1](https://github.com/International-Data-Spaces-Association/InformationModel/commit/d492bc16e93ca1614a3ea4f7efa594747cf277a7).
> My suggestion:
> We need to specify how Time is expected in the IDS, either XSD Datatypes (simple) or TemporalInstances (complex types). We should not define time instances, days of week, months etc. by ourselves.
> 
> _Originally posted by @sebbader in [#273 (comment)](https://github.com/International-Data-Spaces-Association/InformationModel/issues/273#issuecomment-655010415)_

I totally agree with @sebbader! And I recommend to use time-ontology-idea of handling related problems. So, use Instant and Interval.

**Comment by sebbader** [comment]
Hi @jlangkau! Something weird has happened here. Can you find the original topic of this issue? The current title and description confuse me. And please, if possible, add a direct link to the file and line number (see Githubs permalink feature) to directly link the discussion with the correct point in the IM.

**Comment by jlangkau** [comment]
@sebbader : yes, weird! Sorry I can't solve/answer... but: I think it points to the fact, that I like to see more time-ontology stuff in the house (here: those operators)...

**Comment by sebbader** [comment]
Then I have renamed the issue. Let us continue the discussion here, especially as we are currently working also on the IDS-G explanations.
